Tools/AP_Periph/can.cpp

```diff
@@ -483,7 +483,11 @@ static void handle_beep_command(CanardInstance* ins, CanardRxTransfer* transfer)
     fix_float16(req.duration);
     buzzer_start_ms = AP_HAL::native_millis();
     buzzer_len_ms = req.duration*1000;
+#ifdef HAL_PERIPH_ENABLE_BUZZER
     float volume = constrain_float(periph.g.buzz_volume/100.0, 0, 1);
+#elif defined(HAL_PERIPH_ENABLE_NOTIFY)
+    float volume = constrain_float(periph.notify.lib.get_buzz_volume()/100, 0, 1);
```

Comment:
100.0f not 100, also in the above line 100.0 should be 100.0f, we might be forcing needless double operation.

Reply:
fixed